rocAL/rocAL_pybind/amd/rali/pipeline.py

```diff
@@ -230,7 +239,9 @@ def define_graph(self):
         """This function is defined by the user to construct the
         graph of operations for their pipeline.
         It returns a list of outputs created by calling RALI Operators."""
-        raise NotImplementedError
+        print("define_graph NotImplemented")
+        return 0
```

Comment:
return 0 is usually for success. May need error code to indicate not implemented

Reply:
This was added to have trainings run of both the new API & old API simultaneously for time being

Once we confirm that all the trainings using new API is working fine, we will revert this change